pillar/redash.sls

```diff
@@ -1,3 +1,19 @@
+host_id: ocp14
+
+network:
+  netplan: True
+  ipv4:
+    primary_ip: 139.162.199.85
+    primary_ip_subnet_mask: "/24"
+    gateway_ip: 139.162.199.1
+    dns_servers: [ 178.79.182.5, 176.58.107.5, 176.58.116.5, 176.58.121.5, 151.236.220.5, 212.71.252.5, 212.71.253.5, 109.74.192.20, 109.74.193.20, 109.74.194.20 ]
+  ipv6:
+    primary_ip: 2a01:7e00:e000:02cc::14
```

Comment:
Where is this from? I don't see any IP ending in `::14` at https://cloud.linode.com/linodes/30254827/networking

Also, why don't we need to set a `primary_ip_subnet_mask`? It defaults to 128, but I thought our mask was 64:

```
Open a support ticket with Linode to add the new server to our ``2a01:7e00:e000:02cc::/64`` IPv6 range.
```

Reply:
We have all IP addresses under `2a01:7e00:e000:02cc:...` and this is for our entire Linode network - shared across all severs.
I chose `::14` because this is running on OCP14.

I only expect to only require one IPv6 address (`/128`) per server most of the time.

Reply:
Aha, we can update the docs to explain how to assign IP addresses without duplication.